website/blog/2022-11-17-pact-2022-end-of-year-wrapup.md

```diff
@@ -60,38 +67,40 @@ Our first plugin provides support for testing gRPC with Pact, and Pacts core, bu
 
 In the meantime, why not get powered by Pact plugins and create your own?
 
-Find out how to get started <https://docs.pact.io/plugins>
+Find out how to [get started] (https://docs.pact.io/plugins)
```

Comment:
hey hey, shouldn't have a space here between `[]` and `()` for it to render as a link properly 👍🏾